src/boringssl/gen_build_yaml.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, sources):
     def WriteFiles(self, files):
         test_binaries = ['ssl_test', 'crypto_test']
         asm_outputs = {
-            key: value for key, value in files.items() if any(
+            key: value for key, value in list(files.items()) if any(
```

Comment:
I think `list(X.items())` should be `X.items()` when it's being used in the `for-in`.

Reply:
Done.